policyengine_us/parameters/gov/usda/csfp/amount.yaml

```diff
@@ -0,0 +1,10 @@
+description: The following food value amount is provided under the Commodity Supplemental Food Program to each eligible person.
+values:
+  2024-01-01: 600
```

Comment:
Is there a primary government source for this? The closest I found was [this](https://fns-prod.azureedge.us/sites/default/files/resource-files/csfp-program-fact-sheet-2019.pdf) from FNS, showing that:
>An average of almost 676,000 people each month participated in the program in fiscal year (FY) 2018.
and
>For FY 2019, Congress appropriated $222.891 million for CSFP.

Assuming similarity between years, that'd be $330 per year. We also use average government expenditure for valuing WIC and Medicaid.

Reply:
I don't have a primary source for this. I remember Kathleen telling me that she got $50 from someone at the state. I can change it to whatever you want as MFB will just override it on our end if we need to.

Reply:
OK let's go with $330 with that source and a comment explaining